PixArt/nodes.py

```diff
@@ -24,12 +33,28 @@ def INPUT_TYPES(s):
 	CATEGORY = "ExtraModels/PixArt"
 	TITLE = "PixArt Checkpoint Loader"
 
-	def load_checkpoint(self, ckpt_name, model):
+	def load_checkpoint(self, ckpt_name, model, dtype: str):
 		ckpt_path = folder_paths.get_full_path("checkpoints", ckpt_name)
 		model_conf = pixart_conf[model]
+
+		target_dtype: torch.dtype | None = None

```

Comment:
Probably shouldn't use `|` since it doesn't work on older python versions. See also [here](https://github.com/comfyanonymous/ComfyUI/pull/2578)